bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```diff
@@ -906,8 +907,7 @@ def _impl(ctx):
     else:
         fail("Unsupported target platform!")
 
-    # TODO: Need to support non-macOS ARM platforms here.
-    if ctx.attr.target_cpu == "darwin_arm64":
+    if ctx.attr.target_cpu in ["aarch64", "darwin_arm64"]:
         features.append(aarch64_cpu_flags)
```

Comment:
FWIW it feels slightly odd that this is unified as aarch64 here, but arm64  in script_utils.py. But maybe I'm missing something.

Reply:
AFAIK, `aarch64` is more accurate term for the CPU. The `arm64` comes from macOS AFAIK, and isn't the nomenclature that Arm officially uses. It seems reasonable as a suffix to `darwin`, but anywhere else I would hope far `aarch64` to be the canonical name.

Sadly, Go picked `arm64` as *their* canonical name: https://github.com/golang/go/blob/master/src/go/build/syslist.go#L62

And that is why buildifier, buildozer, and the target determinator use `arm64` suffix even on Linux -- they're Go builds. So for anything where we have to match Go or macOS, we should canonicalize to `arm64`. For anything else or anything generic across, we should try to use `aarch64` wherever we can. At least, that's the strategy I've been trying to follow here.